docs/documentation/api.html

```
@@ -944,7 +946,13 @@ <h4>See Also:</h4><p><a href="#giza_render_alpha">giza_render_alpha</a> </p><h3>
 <h4>See Also:</h4><p><a href="#giza_render">giza_render</a> </p><h3><a name="giza_render_gray_float">giza_render_gray_float</a><hr></h3>
 <table class="proto">
 <tr><td width=10%>void</td><td width=20%>giza_render_gray_float</td><td> (int sizex, int sizey, const float* data, int i1, int i2, int j1, int j2, float valMin, float valMax, int extend, int filter, const float *affine) ;</tr></table><p>Same functionality as giza_render_gray but renders in grayscale</p>
-<h4>See Also:</h4><p><a href="#giza_render_gray">giza_render_gray</a> <a href="#giza_render">giza_render</a> </p><h3><a name="giza_draw_pixels">giza_draw_pixels</a><hr></h3>
+<h4>See Also:</h4><p><a href="#giza_render_gray">giza_render_gray</a> <a href="#giza_render">giza_render</a> </p><h3><a name="giza_render_gray_shade">giza_render_gray_shade</a><hr></h3>
+<table class="proto">
+<tr><td width=10%>void</td><td width=20%>giza_render_gray_shade</td><td> (int sizex, int sizey, const double* data, int i1, int i2, int j1, int j2, double fg, double bg, int extend, int filter, const double *affine) ;</tr></table><p>Greyscale image with explicit foreground and background data levels. BG maps to the dark end of the ramp, FG to the light end. When FG < BG the grey ramp is reversed automatically.</p>
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </td> ], start tag match failed [ <td> ] on line 951.

(tag-pair)